data-migration/quick_install_tools.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Quick install guide of TiDB data migration tools
+summary: Learn how to install TiDB data migration tools.
+---
+
+# Migrate from Aurora snapshots to TiDB
+
+This document introduces how to install data migration tools quickly.
+
+## Complete the following tasks before start
+
+- [Hardware and software requirements](/hardware-and-software-requirements.md)
+- [Environment and system configuration check](/check-before-deployment.md)
+
+## Install TiUP
+
+Starting with TiDB 4.0, TiUP, as the package manager, helps you easily manage different cluster components in the TiDB ecosystem. Now you can run any component with only a single TiUP command line.
```

Comment:
```suggestion
Since TiDB 4.0, TiUP serves as the package manager to help you easily manage different cluster components in the TiDB ecosystem. Now you can manage any components with only one single TiUP command line.
```